### DISCUSSION TITLE
Loading extension doesn't seem to load javascript?

### DISCUSSION THREAD

**Discussion by derekantrican** [cefsharp/CefSharp#3392] [Q&A] [answered]
I'm trying to load an extension referencing [the example](https://github.com/cefsharp/CefSharp/blob/master/CefSharp.WinForms.Example/BrowserForm.cs) and it seems to load in (popup opens and everything) but it doesn't seem to fully load in everything. I think it's missing the extension's javascript & some other resources? Here's a comparison screenshot of the popup between CEFSharp & my actual browser (both are loading from the exact same folder on my computer):

![image](https://user-images.githubusercontent.com/1558019/106957149-7f5c4300-66ec-11eb-951c-6e41d96efb46.png)

Any ideas what I might be missing?

**Comment by amaitland** [comment, marked as answer]
CEF only supports a subset of the extension API. I expect the JavaScript has errors as it's accessing APIs that aren't implemented. Extension support is very limited, there is nothing I can do about that. 

See https://bitbucket.org/chromiumembedded/cef/issues/1947/add-support-for-chrome-extensions

**Reply by derekantrican** [reply]
Gotcha. Is there a way to see the console or other errors/output from the extension? If it's not core functionality I could try to strip the failing parts out

**Reply by derekantrican** [reply]
I found https://stackoverflow.com/questions/29117882/debugging-javascript-in-chromium-embedded-framework - I'll mess around with that a bit. Thanks for the help

**Reply by amaitland** [reply]
Check the log file see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
